src/components/accordion/index.js

```diff
@@ -1,21 +1,36 @@
-import angular from 'core/angular';
-import './Accordion.scss';
+import angular from "core/angular";
+import "./Accordion.scss";
+class AccordionComponentController {
+    imageuri;
```

Comment:
I think that would be a good practice to use camelCase when we name variables. Some articles shows that camelCase it is a definition used to prevent misspelling on composed names.

https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c.

What do you think about it?